notebooks/community/vizier/conversions_vertex_vizier_and_open_source_vizier.ipynb

```diff
@@ -0,0 +1,621 @@
+      "source": [
+        "import datetime\n",
+        "import math"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "CWuu4wmki2X3"
+      },
+      "source": [
+        "## Tutorial\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KyEjqIdnad0w"
+      },
+      "source": [
+        "This section defines some parameters to create the study and optimize the objective function.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "8HCgeF8had77"
+      },
+      "outputs": [],
+      "source": [
+        "# These will be automatically filled in.\n",
+        "STUDY_DISPLAY_NAME = \"{}_study_{}\".format(\n",
+        "    PROJECT_ID.replace(\"-\", \"\"), datetime.datetime.now().strftime(\"%Y%m%d_%H%M%S\")\n",
+        ")\n",
+        "\n",
+        "print(\"REGION: {}\".format(REGION))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8NBduXsEaRKr"
+      },
+      "source": [
+        "### Define the parameters\n",
+        "\n",
+        "The following is a sample study configuration, built as a hierarchical python dictionary. It is already filled out. Run the cell to configure the study.\n",
+        "\n",
+        "__`USE_VERTEX_VIZIER`__: Uses Vertex Vizier SDK to do the optimization if True. Use OSS Vizier otherwise.\n",
+        "\n",
+        "__`SUGGESTION_COUNT`__: The number of suggestions (trials) requested in a single request.\n",
+        "\n",
+        "__`MAX_NUM_ITERATIONS`__: The number of iterations to explore before stopping. It is set to 4 to shorten the time to run the code, so don't expect convergence. For convergence, it would likely need to be about 20.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "E1VNJ4YBznhR"
+      },
+      "outputs": [],
+      "source": [
+        "USE_VERTEX_VIZIER = True  # @param {type:\"boolean\"}\n",
+        "\n",
+        "MAX_NUM_ITERATIONS = 4  # @param {type:\"integer\"}\n",
+        "\n",
+        "SUGGESTION_COUNT = 2  # @param {type:\"integer\"}\n",
+        "\n",
+        "OWNER = \"owner\"  # @param {type:\"string\"}\n",
+        "\n",
+        "SERVICE_ENDPOINT = \"127.0.0.1:8888\"  # @param {type:\"string\"}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "4_Yvt-7Z8_re"
+      },
+      "source": [
+        "### Import the package and define CreateStudy for different sources\n",
+        "\n",
+        "In Vertex Vizier, `project` and `location` are specified and `Study.create_or_load` is called to create a study. In OSS Vizier, you'll need to input the owner of your study and the server address with format [ip:port]. To bring up the OSS Vizier server, please follow the [instruction](https://oss-vizier.readthedocs.io/) on the OSS Vizier website."
```

Comment:
are specified -> are already specified.

In OSS Vizier, you'll need to... -> you need to

with format -> in the format

the instruction -> the instructions